drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/import_concern.rb

```diff
@@ -205,20 +213,22 @@ def self.fix_date_format(string)
       d.strftime('%Y-%m-%d')
     end
 
+    # returns a TimeWithZone instead of a string so that we have
+    # more info for things like ActiveRecord::Import which wont know
+    # the intended timezone otherwise
     def self.fix_time_format(string)
       return unless string
 
       string = string.gsub('/', '-')
       # Ruby handles yyyy-m-d just fine, so we'll allow that even though it doesn't match the spec
       return string if /\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}/.match?(string)
-      return string.to_time.strftime('%Y-%m-%d %H:%M:%S') if /\d{4}-\d{1,2}-\d{1,2} \d{1,2}:\d{1,2}:?\d{0,2}?/.match?(string)
+      return Time.zone.parse(string) if /\d{4}-\d{1,2}-\d{1,2} \d{1,2}:\d{1,2}:?\d{0,2}?/.match?(string)
 
       # Sometimes times come in mm-dd-yyyy hh:mm
       if /\d{1,2}-\d{1,2}-\d{4} \d{1,2}:\d{1,2}:?\d{0,2}?/.match?(string)
         date, time = string.split(' ')
         month, day, year = date.split('-')
-
-        return "#{year}-#{month.rjust(2, '0')}-#{day.rjust(2, '0')} #{time}".to_time.strftime('%Y-%m-%d %H:%M:%S')
+        return Time.zone.parse("#{year}-#{month.rjust(2, '0')}-#{day.rjust(2, '0')} #{time}")
```

Comment:
I think this isn't picking the right timezone.  On the left, previous behavior, on the right, current behavior (loaded on top, imported on bottom)
![Screen Shot 2021-02-03 at 9 22 31 AM](https://user-images.githubusercontent.com/1346876/106760256-97a36380-6601-11eb-8b36-a2d11d33c6cf.png)